src/shared/lib/svgToPdfDownload.ts

```diff
@@ -26,7 +26,7 @@ export default function svgToPdfDownload(fileName: string, svg: any) {
     const pdf = new jsPDF(direction, 'pt', [width, height]);
 
     pdf.addFileToVFS("FreeSans-normal.ttf", font.FreeSans);
-    pdf.addFont("FreeSans-normal.ttf", "FreeSans", "normal");
+    pdf.addFont("FreeSans-normal.ttf", "Arial", "normal");
```

Comment:
Lets put a comment above this explaining why were are doing this slightly strange thing

Reply:
@alisman Sure. Also removed the unused functions (we don't use promise anymore, they are not referenced).